### PR TITLE
Add administrator role & description

### PR DESCRIPTION
**Pull request by realChainLife**
This information will be required for targeted grants.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,10 +31,11 @@ RFPs are grants for specific development work. As the name suggests, we are requ
 
 OPEN RFPs:
 
-[**Chainlink x Filecoin**](https://github.com/filecoin-project/devgrants/blob/master/rfps/chainlink-and-filecoin.md) -- Deadline September 17
 
 [**Near Protocol x Filecoin**](https://github.com/filecoin-project/devgrants/blob/master/rfps/near-and-filecoin.md) -- Deadline October 30 
 
+[**Chainlink x Filecoin**](https://github.com/filecoin-project/devgrants/blob/master/rfps/chainlink-and-filecoin.md) -- Open until December 2021
```
Comment (mishmosh) [inline README.md:37]:
```suggestion
[**Chainlink x Filecoin**](https://github.com/filecoin-project/devgrants/blob/master/rfps/chainlink-and-filecoin.md) -- Rolling submissions, open until December 2021
```

File: .github/ISSUE_TEMPLATE/opengrant.md [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 **Proposer:** `replace with your GitHub username`
 
+**Administrator:** `replace with pre-selected admin for targeted grants`
```
Comment (mishmosh) [inline .github/ISSUE_TEMPLATE/opengrant.md:9]: Most applicants won't know what an Administrator is. Can we change the wording so this feels more optional, and doesn't scare away people who are submitting a non-sponsored open grant?
Maybe this:

**(Optional) Technical Sponsor:** `if you have previously discussed this project with a member of the IPFS or Filecoin project teams, include their name and/or github handle here`

**Review by mishmosh — COMMENTED** [review]
Suggested a wording change in Line 9 to reduce confusion for open grant applicants. Otherwise lgtm!

File: .github/ISSUE_TEMPLATE/opengrant.md (hunk 2 above)
Comment (realChainLife) [reply .github/ISSUE_TEMPLATE/opengrant.md]: Awesome, thanks Mosh

**Pull request merged**